src/app/static/src/app/components/Translated.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <span>{{text}}</span>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import i18next from "i18next";
+    import {mapState} from "vuex";
+    import {RootState} from "../root";
+
+    export default Vue.extend({
+        name: "Translated",
+        props: ["value"],
```

Comment:
I don't love "value" as the prop name here - it's really the key of the translations dictionary, but "key" is a reserved word. Any ideas?

Reply:
Yeah, that's a bit confusing. I think some variant of 'key' would be better than 'value'. 'stringKey'?

Reply:
Cool, I'll go with textKey